src/Pos/Wallet/Web/Api.hs

```diff
@@ -257,20 +256,13 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
-type GetHistory =
-       "txs"
-    :> "histories"
-    :> CCapture "walletId" CAccountId
-    :> QueryParam "skip" Word
-    :> QueryParam "limit" Word
-    :> WRes Get ([CTx], Word)
-
 type SearchHistory =
        "txs"
     :> "histories"
-    :> CCapture "walletId" CAccountId
-    :> Capture "search" Text
-    :> QueryParam "account" (CId Addr)
+    :> QueryParam "walletId" (CId Wal)
+    :> QueryParam "accountId" CAccountId
+    :> QueryParam "address" (CId Addr)
+    :> QueryParam "search" Text
```

Comment:
@Pastafarianist unfortunately, purescript-bridge is not as tightly interconnected with haskell code, and you won't detect mismatches in server api and bridge methods in compile time. 
`src/Daedalus/BackendApi.purs:184` contains `searchHistory` method, which now doesn't comply with modified API method in Haskell code.